cypress/e2e/deputies_tab.cy.js

```diff
@@ -23,12 +23,18 @@ describe("Deputies Tab", () => {
             );
         });
 
-        it("shows a dash if no ECM", () => {
+        it("shows a dash if no assurance visit", () => {
```

Comment:
Could we add a happy path test to show the assurance visit data renders correctly? This information would need adding in the db.json file, happy to go through it with you.